scripts/generate_samples.ts

```diff
@@ -0,0 +1,88 @@
+import fs from "fs";
```

Comment:
```suggestion
import fs from "node:fs";
```

Becoming the preferred style for imports that are part of nodejs and not coming from npm, for some reason (something about interoperability).